MAINTAINERS.yml

```diff
@@ -1031,6 +1031,24 @@ Documentation:
     labels:
         - "area: Wifi"
 
+Xen Platform:
```

Comment:
The location of this entry in the MAINTAINERS file is somewhat questionable, but we have no concrete rule about how the areas should be sorted -- we can clean this up later.

Reply:
Actually, I was thinking about this when I change to `Xen Platform` but totally had no idea where to put it. I can move it to a proper location if you already had some suggestions :)